pkg/metrics/metrics.go

```diff
@@ -49,5 +50,12 @@ func CreateMetrics(reg prometheus.Registerer) Metrics {
 			},
 			[]string{"result"},
 		),
+		EC2DescribeInstanceCallCount: factory.NewCounter(
+			prometheus.CounterOpts{
+				Namespace: Namespace,
+				Name:      "ec2_describe_instances_call",
```

Comment:
should be "ec2_describe_instances_calls" (plural) or "ec2_describe_instances_calls_total" according to https://prometheus.io/docs/practices/naming/

Reply:
prefer ec2_describe_instances_calls_total cuz it is an 'accumulating count'

Reply:
Yeah I think it should be `_total` if we go with the prometheus guide.  E.g.

```
process_cpu_seconds_total (for an accumulating count with unit)
```

This will match user's expectations.